Wrapping/Python/Packaging/setup.py.in

```diff
@@ -1,84 +0,0 @@
-import sys
-
-if "setuptools" in sys.modules.keys():
-    from setuptools import setup, Distribution
-else:
-    from distutils.core import setup
-    from distutils.dist import Distribution
-
-import re
-
-doc_files = @SimpleITK_DOC_FILES_AS_LIST@
-
-def get_pep386version():
```

Comment:
Instead, would it make sense to use something like https://github.com/warner/python-versioneer 

It automatically generate the version based on the last tag.

Here is an example of configuration: https://github.com/scikit-build/cmake-python-distributions/blob/3e78241af63ea0c08171e7da032f92a43fe6bbff/setup.cfg#L19-L24